boards/BMS/Src/App/states/App_FaultState.c

```diff
@@ -24,14 +24,12 @@ static void FaultStateRunOnTick100Hz(struct StateMachine *const state_machine)
 {
     App_AllStatesRunOnTick100Hz(state_machine);
 
-    struct BmsWorld *        world       = App_SharedStateMachine_GetWorld(state_machine);
-    struct Airs *            airs        = App_BmsWorld_GetAirs(world);
-    struct ErrorTable *const error_table = App_BmsWorld_GetErrorTable(world);
+    struct BmsWorld *world = App_SharedStateMachine_GetWorld(state_machine);
+    struct Airs *    airs  = App_BmsWorld_GetAirs(world);
 
-    bool is_error_table_cleared = !App_SharedErrorTable_HasAnyAirShutdownErrorSet(error_table);
-    bool is_air_negative_open   = !App_Airs_IsAirNegativeClosed(airs);
+    bool is_air_negative_open = !App_Airs_IsAirNegativeClosed(airs);
 
-    if (is_error_table_cleared && is_air_negative_open)
+    if (is_air_negative_open)
```

Comment:
There is a even simpler fix. Moving the call to `App_AllStatesRunOnTick100Hz(state_machine);` to after this check allows all-states to over-write the next-state to fault state if there are any active faults. I can't see a reason why this change would cause any issues but let me know if it does!

Reply:
Hmm that's a little too jank. Can you just check `App_Accumulator_CheckFaults` and `App_TractveSystem_CheckFaults` here instead?

Reply:
I could, but that would meant that `App_Accumulator_CheckFaults` and `App_TractiveSystem_CheckFaults` would be getting called twice inside `FaultStateRunOnTick100Hz`. If we can afford that then sure

Reply:
Is the concern performance? It's not ideal, but shouldn't be significant. 
We call the `AllStatesRunOnTick` function first in every state, so let's keep doing so, to avoid any unexpected behavior down the line.

Reply:
Okay so long as it's not a performance concern that sounds like a good idea